src/ready.js

```diff
@@ -0,0 +1,72 @@
+// DOM ready check in plain js taken from https://github.com/jfriend00/docReady
+(function(funcName, baseObj) {
```

Comment:
it looks like there's an NPM module that could be used instead? https://www.npmjs.com/package/domready